{
  "CVE Code": "CVE-2022-34265",
  "Published": "2022-07-04T00:00:00.000Z",
  "Updated": "2025-02-13T16:32:45.198Z",
  "Description": "An issue was discovered in Django 3.2 before 3.2.14 and 4.0 before 4.0.6. The Trunc() and Extract() database functions are subject to SQL injection if untrusted data is used as a kind/lookup_name value. Applications that constrain the lookup name and kind choice to a known safe list are unaffected.",
  "ProblemTypes": [
    "n/a"
  ],
  "Affected": [
    {
      "vendor": "n/a",
      "product": "n/a",
      "versions": [
        "n/a"
      ]
    }
  ],
  "Reference": [
    "https://groups.google.com/forum/#%21forum/django-announce",
    "https://docs.djangoproject.com/en/4.0/releases/security/",
    "https://www.djangoproject.com/weblog/2022/jul/04/security-releases/",
    "https://security.netapp.com/advisory/ntap-20220818-0006/",
    "https://www.debian.org/security/2022/dsa-5254",
    "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/HWY6DQWRVBALV73BPUVBXC3QIYUM24IK/",
    "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/LTZVAKU5ALQWOKFTPISE257VCVIYGFQI/"
  ]
}